tests/expected_records.db

```diff
@@ -1,7 +1,7 @@
 # This file was automatically generated on Fri 13 Aug 2021 16:49:03 BST.
-# 
+#
 # *** Please do not edit this file: edit the source file instead. ***
-# 
+#
```

Comment:
Hmmm.  Something fishy about line 3 in this context!

Reply:
Good point, will remove it